ArduCopter/Copter.h

```diff
@@ -589,13 +589,13 @@ class Copter : public AP_Vehicle {
         ESCCAL_DISABLED = 9,
     };
 
-    enum Failsafe_Action {
-        Failsafe_Action_None           = 0,
-        Failsafe_Action_Land           = 1,
-        Failsafe_Action_RTL            = 2,
-        Failsafe_Action_SmartRTL       = 3,
-        Failsafe_Action_SmartRTL_Land  = 4,
-        Failsafe_Action_Terminate      = 5
+    enum class Failsafe_Action : int8_t {
```

Comment:
We've traditionally used upper-case for the actual enumeration class values.

It's not universal, but the vast majority follow that pattern.

Underscore should also be removed, so `FailsafeAction`, not `Failsafe_Action`.  You can browse through the code and find that that too is prevalent.

See `FailsafeOption` just down below, for example!

Also, this should be `uint8_t`, not `int8_t`.